lib/terminology_table.erb

```diff
@@ -33,7 +33,10 @@
             <% binding['terminology'].each do |terminology| %>
               <% unless terminology['system'].nil? && terminology['display'].nil? %>
                   <li>
-                    <% if terminology['display'] && terminology['system'] %>
+                    <% #TODO: Adjust displays to be consistent once all resourced differentiate system from info url %>
+                    <% if terminology['display'] && terminology['system'] && terminology['info_link'] %>
+                      <%= "<a target=\"_blank\" href=\"#{terminology['info_link']}\">#{terminology['display']}</a> (#{terminology['system']})" %>
```

Comment:
For the first sweep here, I'm doing something similar to what is the "future" idea, minus the systems header. It will have a Details: ... (to differentiate from the description above) and System:... 
No bullets yet until I can refactor this once all the resources follow this pattern. Once I check in, you can see what you think